src/platform/lumin-runtime/elements/builders/quad-builder.js

```diff
@@ -170,36 +170,36 @@ export class QuadBuilder extends RenderBuilder {
     const absolutePath = this.getPropertyValue('absolutePath', false, properties);
     const descriptor = this.getPropertyValue('descriptor', -1, properties);
     const basePath = this.getPropertyValue('basePath', null, properties);
-    return prism.createAnimationResourceId(properties.fileName, absolutePath, descriptor, basePath);
+    return this._createNode(prism, 'createAnimationResourceId', properties.fileName, absolutePath, descriptor, basePath);
```

Comment:
Maybe consider renaming _createNode if it's going to be used for more than just Nodes

Reply:
You're right! I was just lazy :(